src/components/sidebar/tabs/BaseWorkflowsSidebarTab.vue

```
@@ -0,0 +1,352 @@
+          <template #node="{ node }">
+            <WorkflowTreeLeaf :node="node" />
+          </template>
+        </TreeExplorer>
+      </div>
+    </template>
+  </SidebarTabTemplate>
+  <ConfirmDialog />
+</template>
+
+<script setup lang="ts">
+import ConfirmDialog from 'primevue/confirmdialog'
```

ESLint

[error] Unable to resolve path to module 'primevue/confirmdialog'.

(import-x/no-unresolved)